physionet-django/events/forms.py

```diff
@@ -72,7 +73,18 @@ class EventDatasetForm(forms.ModelForm):
     """
     A form for adding datasets to an event.
     """
-    dataset = forms.ModelChoiceField(queryset=PublishedProject.objects.all(),
+
```

Comment:
These changes do not (as far as I can see) relate to "Adding cohost permissions to view/respond to participation requests".

Reply:
The reason as mentioned above.